shared/logger/index.js

```diff
@@ -121,14 +120,18 @@ const devLoggers = () => ({
 
 const prodLoggers = () => ({
   action: isMobile
-    ? new NativeLogger()
-    : new DumpPeriodicallyLogger(new RingLogger(500), 10 * 60e3, writeLogLinesToFile, 'Action'),
-  debug: new NullLogger(),
+    ? new RingLogger(200)
+    : new DumpPeriodicallyLogger(new RingLogger(200), 10 * 60e3, writeLogLinesToFile, 'Action'),
+  debug: new RingLogger(500),
   error: isMobile
     ? new NativeLogger()
     : new DumpPeriodicallyLogger(new RingLogger(100), 1 * 60e3, writeLogLinesToFile, 'Error'),
-  info: new RingLogger(100),
-  warn: new RingLogger(100),
+  info: isMobile
+    ? new NativeLogger()
+    : new DumpPeriodicallyLogger(new RingLogger(500), 1 * 60e3, writeLogLinesToFile, 'Info'),
```

Comment:
error only has a ring size of 100, but info and warn has ring sizes of 500. Shouldn't error be at least as important as info/warn, and have a ring size of 500 or larger?

Reply:
yep! my reasoning was that in practice we use warn a lot more than error, but that sounds fine to me